cernopendata/modules/records/serializers/schemas/schemaorg_schemas.py

```diff
@@ -0,0 +1,149 @@
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# CERN Open Data Portal is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with CERN Open Data Portal; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+"""CERN Opendata schema.org marshmallow schema."""
+
+from __future__ import absolute_import, print_function
+
+from marshmallow import Schema, fields
+
+from marshmallow.decorators import post_dump
+
+
+class RecordSchemaorgSchema(Schema):
+    """schema.org JSON-LD schema for CERN Opendata Record."""
+
+    # JSON-LD minimum requirements
+    context_ = fields.Field(default='https://schema.org/', dump_to='@context')
+    id_ = fields.Method('get_identifier', dump_to='@id')
+
+    # Minimum functional
+    type_ = fields.Method('get_type', dump_to='@type')
+    name = fields.Str(attribute='title', dump_to='name')
+    description = fields.Method('get_description', dump_to='description')
+    identifier = fields.Method('get_identifier', dump_to='identifier')
+    url = fields.Method('get_url', dump_to='url')
+    creator = fields.Method('get_creator', dump_to='creator')
+    date_created = fields.Str(attribute='date_published',
+                              dump_to='dateCreated')
+    date_published = fields.Str(attribute='date_published',
+                                dump_to='datePublished')
+    publisher = fields.Method('get_publisher', dump_to='publisher')
+
+    # Minimum operational
+    # license = fields.Str('license', dump_to='license')
+    # data_standard = fields.Str('dataStandard', dump_to='dataStandard')
+    # date_modified = fields.Str('dateModified', dump_to='dateModified')
+    # structure = fields.Str('structure', dump_to='structure')
+    # access_url = fields.Str('accessUrl', dump_to='accessUrl')
+    # access_interface = fields.str('accessInterface',
+    #                              dump_to='accessInterface')
+
+    def get_creator(self, obj):
+        """Get creator(s) based on authors or collaboration field."""
+        authors = obj.get('authors', None)
+
+        if authors:
+            return [{"@type": "Person", 'name': x['name']} for x in authors]
+        else:
+            # Note: Can there be many Organizations in a Record?
+            #       Now only one is expected.
+            collaboration = obj.get('collaboration', None).get('name', None)
```

Comment:
`None` does not have `get`, so this could fail if the `collaboration` key is not in the object.  It should be safer to use:

```python
collaboration = obj.get('collaboration', {}.get('name', None)
```

Also, instead of the second `None`, maybe empty string might be clearer there... as if indicating the usual "output type", as it were...